cmd/scollector/collectors/redis_unix.go

```diff
@@ -286,7 +286,10 @@ func redisInit() {
 			}
 			port := strings.Split(sp[2], ":")[1]
 			if port != "0" {
-				add(port)
+				_, err := util.Command(time.Second, nil, "grep", "-q", "cpu:/docker", fmt.Sprintf("/proc/%s/cgroup", sp[0]))
```

Comment:
I was able to get it to work but it will add a dependency on bash (which I'm fine with). Seems to work testing on ny-intdiscourse01 and ny-gbraylx01, and it's failure mode is to default to be enabled.

If someone is using namespaces but the redis instance is still accessible via localhost then this will cause redis to not be monitored, where it was previously so long as the connection to :6379 or whatever the port was worked. I'm also fine with addressing this in the future if that is ever reported.